flang/test/Lower/OpenMP/empty-omp-parallel.f90

```diff
@@ -10,24 +10,32 @@
 program parallel
 
         integer :: a,b,c
+        integer num_threads
```

Comment:
Do you want to add `::` to match the previous declaration style?

Reply:
Ah, I missed that again! I'll take care of this in subsequent revision or before squashing.